great_expectations/validator/validator.py

```diff
@@ -1565,10 +1554,16 @@ def save_expectation_suite(
             )
 
     # TODO: <Alex>Should "include_config" also be an argument of this method?</Alex>
+    @public_api
+    @deprecated_argument(
```

Comment:
```suggestion
    @deprecated_argument(  # noqa: C901
```
It's possible you'll need it here too but hopefully not 😬